api/app/serializers/spree/api/v2/platform/product_serializer.rb

```diff
@@ -0,0 +1,69 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class ProductSerializer < BaseSerializer
+          include ::Spree::Api::V2::DisplayMoneyHelper
+
+          set_type :product
+
+          attributes :name, :description, :available_on, :slug, :meta_description, :meta_keywords, :updated_at
+
+          attribute :purchasable do |product|
+            product.purchasable?
+          end
+
+          attribute :in_stock do |product|
+            product.in_stock?
+          end
+
+          attribute :backorderable do |product|
+            product.backorderable?
+          end
+
+          attribute :available do |product|
+            product.available?
+          end
+
+          attribute :currency do |_product, params|
+            params[:currency]
+          end
```

Comment:
@MatthewKennedy for Platform API you only need to add `include ::Spree::Api::V2::ResourceSerializerConcern`

Reply:
I'll go over this again, and get it in good shape.